src/lib/DataTableHeader/DataTableRow.tsx

```diff
@@ -22,14 +23,54 @@ export function DataTableRow<T, TRouteNames>({
   columns,
   actions,
   rowStyle,
+  rowHighlight,
   actionsPosition,
 }: DataTableRowProps<T, TRouteNames>) {
   const keyValue = getDeepValue(record, keyField);
   const [collapsed, setCollapsed] = useState(true);
 
+  var operator_table = {
+    ">": function (a: number | Date, b: number | Date) {
+      return a > b;
+    },
+    "<": function (a: number | Date, b: number | Date) {
+      return a < b;
+    },
+    "==": function (a: number | Date, b: number | Date) {
+      return a == b;
+    },
+    "!=": function (a: number | Date, b: number | Date) {
+      return a != b;
+    },
+  };
+
+  function setStyle(rowObjectT: T, rowHighlight?: RowHighlightInterface<T>): CSSProperties | undefined {
+    const defaultStyle: CSSProperties = {
+      backgroundColor: "rgba(255,0,0,0.7)",
+      color: "white",
+    };
+
+    if (rowHighlight) {
```

Comment:
Personally, I prefer guard clauses. Because it reduces the complexity of the code. 

https://deviq.com/design-patterns/guard-clause

But it's personal taste, so you don't have to change it if you disagree :-)

Reply:
Yes, it's better like this, fixed!